README.md

```diff
@@ -5,9 +5,23 @@ This is a lab for investigating adding a layer of webservices to Joomla
 
 For more details join #webservices channel at https://glip.com/
 
-
 How to install
 ====
+* Install a full current Joomla somewhere
+* git clone https://github.com/joomla-projects/webservices.git OR https://github.com/chrisdavenport/webservices.git
+* Zip or tar-gzip everything in the /component directory
+* Install component package file as an extension in the usual way
+* Copy all the files from the github clone to the web root of the CMS
+* sudo composer require willdurand/Negotiation (don't know why this is needed, but it is)
```

Comment:
Okay, so I must have missed a step when I built the composer.lock in the webservices repo.  I'll check that out next time I do a full build.